Package.swift

```diff
@@ -7,10 +7,10 @@ let package = Package(
     name: "RollbarSDK",
     platforms: [
         // Oldest targeted platform versions that are supported by this product.
-        .macOS(.v10_10),
+        .macOS(.v10_11),
         .iOS(.v9),
         .tvOS(.v11),
-        .watchOS(.v4),
+        .watchOS(.v7),
```

Comment:
Is this bump necessary? Ie. are we telling devs they can only use the SDK with watchOS v7 and macOS 10.11 and up?

Reply:
yes, some OOM detection-related Apple APIs which we call are only available starting watchOS v7. 
older clients can always use the last SDK version release that did not support the OOM detection.